plugin-name/internals/Transient.php

```diff
@@ -40,12 +41,12 @@ public function transient_caching_example() {
 		// Use wp-cache-remember package to retrive or save in transient
 		return remember_transient(
             $key,
-            function () use ( $key ) {
+            function () {
 				// If there's no cached version we ask
 				$response = wp_remote_get( 'https://jsonplaceholder.typicode.com/todos/' );
 				if ( is_wp_error( $response ) ) {
-					// In case API is down we return the last successful count
-					return;
+					// In case API is down we return an empty object
```

Comment:
I haven't found a way to return meaningful data when the API is down.

Reply:
I think that is good enough, after all is an example.